DigitalLearningSolutions.Web/appsettings.json

```diff
@@ -30,5 +30,10 @@
     "ClientCode": "DigitalLearningSolutions"
   },
   "ExcelPassword": "0f8cc6f0-9f21-4f0f-9cb9-365675490458",
-  "IsTransactionScope": true
+  "IsTransactionScope": true,
+  "LearningHubReportAPIConfig": {
+    "BaseUrl": "https://reportapi-test.test-learninghub.org.uk",

```

Comment:
Could we replace this with the live equivalent to reduce the config changes required during deployment?

Reply:
Replaced with the production URL @kevwhitt-hee